### DISCUSSION TITLE
Default Game and Mod for Exult Studio

### DISCUSSION THREAD

**Discussion by KnightCaptainU7** [exult/exult#439] [General]
How do I set both the default game AND the mod when opening Exult Studio? I'd like to have it assume that the mod I'm working on is my own.

**Comment by DominusExult** [comment]
When you start Exult Studio from Exult it will automatically use the game+mod you are using in Exult. If you start it independently you have to choose it in the `File -> open game` menu. Or you use the command line options http://exult.info/studio.php#command_line

**Reply by KnightCaptainU7** [reply]
Is there an Exult.cfg entry that matches what modname does via command line?
